features/facebookImport/src/model/analysis.js

```diff
@@ -173,13 +173,13 @@ export async function runAnalysis(analysisClass, enrichedData) {
         const status = await subAnalysis.analyze(enrichedData);
         return {
             analysis: subAnalysis,
-            status: status || createSuccessStatus(analysisClass),
+            status: status || createSuccessStatus(),
```

Comment:
This is probably within scope, in this case I would really prefer if we created a class and instantiate it with these values.

Reply:
Now the test code for analyses depends on this being an array. After we unify the statuses for the analyses and imports, we can also have dedicated objects to represent the returned values for both.

Reply:
Actually I added now `AnalysisExecutionResult` to group those values.